### DISCUSSION TITLE
v2/zh/cookbook/tutorial/command

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#350] [Announcements]
# v2/zh/cookbook/tutorial/command

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.gitee.io/v2/zh/cookbook/tutorial/command.html

**Comment by HF10** [comment]
pnpm add vuepress@next vuepress-theme-hope@next && pnpm i && pnpm up
更新后会显示 points to missing source files
